slackbot_using_azure_openai/service.bal

```diff
@@ -0,0 +1,84 @@
+import ballerina/crypto;
+import ballerina/http;
+import ballerinax/azure.openai.chat;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string slackSigningSecret = ?;
+
+const API_VERSION = "2023-03-15-preview";
+const MAX_MESSAGES = 25;
+
+final chat:Client azureOpenAI = check new (
+    config = {auth: {apiKey: openAIToken}},
+    serviceUrl = serviceUrl
+);
+
+type ChatMessage record {|
+    string role;
```

Comment:
```suggestion
    Role role;
```

Reply:
Addressed in [fdf05f7](https://github.com/ballerina-guides/ai-samples/pull/64/commits/fdf05f7bd315dfff7335c65ce0ce778fb6299c9f)